js/index.js

```diff
@@ -39,4 +39,46 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+logo.src = 'img/logo.png';
+
+const nav = document.querySelectorAll('nav');
+const aTag = document.querySelectorAll('a');
+const ctaSec = document.querySelectorAll('.cta');
+const mainCon = document.querySelectorAll('.main-content');
+const ctaImg = document.getElementById("cta-img");
+const midImg = document.getElementById('middle-img');
+const contact = document.querySelectorAll('.contact');
+const foot = document.querySelectorAll('footer');
+const btn = document.querySelectorAll('button');
+const h1 = document.querySelectorAll('h1');
+const h4 = document.querySelectorAll('h4');
+const p = document.querySelectorAll('p');
+
+
+aTag[0].textContent = 'Services';
```

Comment:
For changing the text content, the purpose of the assignment is to access the siteContent object and change the DOM that way, for example:
```suggestion
navLinks[0].textContent = siteContent["main-content"]["features-h4"]
```
If you notice the syntax, you can see that we're accessing the `siteContent` object with the main-content and features-h4 keys, and that will return whatever the string is, which prevents you from needing to type out all of the text in the assignment. Please let me know if you have any questions about how this works :+1: